slashing-protection/src/main/resources/migrations/postgresql/V00012__add_highwatermark_metadata.sql

```diff
@@ -0,0 +1,63 @@
+ALTER TABLE metadata
+    ADD COLUMN high_watermark_epoch NUMERIC(20),
+    ADD COLUMN high_watermark_slot NUMERIC(20);
```

Comment:
Hmmm, true but if you only set one doesn't that mean we must calculate the other? If so, then I was hoping to burden the user with that since it's an advanced use case and kind of a one-off.

Reply:
That's true you would need to calculate it. And since this is an advanced use case think you are right we just keep this simple and let the user supply the values.